packages/create/src/cli/add/mockserver-config.ts

```diff
@@ -0,0 +1,96 @@
+import { spawnSync } from 'child_process';
+import type { Command } from 'commander';
+import { prompt } from 'prompts';
+import { getWebappPath } from '@sap-ux/project-access';
+import { generateMockserverConfig, getMockserverConfigQuestions } from '@sap-ux/mockserver-config-writer';
+import type { MockserverConfig } from '@sap-ux/mockserver-config-writer';
+import { getLogger, traceChanges, setLogLevelVerbose } from '../../tracing';
+import { validateBasePath } from '../../validation';
+import { relative } from 'path';
+
+/**
+ * Add the "add mockserver config" command to a passed command.
+ *
+ * @param cmd - commander command for adding mockserver config command
+ */
+export function addAddMockserverConfigCommand(cmd: Command): void {
+    cmd.command('mockserver-config [path]')
+        .option('-i, --interactive', 'ask for config options, otherwise use defaults')
```

Comment:
Use i18n texts?

Reply:
I18N is on my list to do.

Reply:
i18n is used now in `mockservice-config-writer`. For cli we do not add i18n.